nncf/tensorflow/quantization/functions.py

```diff
@@ -53,3 +55,33 @@ def _fake_quant_with_min_max_vars(inputs, min_var, max_var, num_bits, narrow_ran
             inputs, min_var, max_var, num_bits=num_bits, narrow_range=narrow_range)
     return tf.quantization.fake_quant_with_min_max_vars(
         inputs, min_var, max_var, num_bits=num_bits, narrow_range=narrow_range)
+
+
+def asymmetric_range_initialization(min_values,
```

Comment:
The function name is not a verb, please rename. Type hints and docstrings are missing, here and below.

Reply:
I fixed it.